### PR TITLE
Add a way to log out

### PR DESCRIPTION
**Pull request by elarb**
This PR introduces the long awaited feature enabling users to logout

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -317,12 +326,26 @@
           el.user = user;
         }
       }
+      function showLogoutButtons() {
+        for (const node of logoutButtons) {
+          node.style.display = 'block';
```
Comment (TimvdLippe) [inline index.html]: Let's add a class rather than inline style. E.g. `node.classList.add('signed')` or something

**Comment by elarb** [comment]
Test

File: src/lancie-login/lancie-logout-button.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        return 'lancie-logout-button';
+      }
+
+      static get properties() {
+        return {
+          labeled: {
+            type: Boolean,
+            value: false,
+          }
+        }
```
Comment (martijnjanssen) [inline src/lancie-login/lancie-logout-button.html]: Missing semicolumn here: causing the build to fail
```suggestion
        };
```

File: src/lancie-login/lancie-logout-button.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    <lancie-ajax id="ajaxLogout" refurl="logout" method="POST" on-lancie-ajax="onLogoutResponse"></lancie-ajax>
+    <div on-tap="tryLogout">
+      <paper-icon-button icon="lancie:exit-to-app"></paper-icon-button>
+      <span hidden="[[!labeled]]">Sign out</span>
+    </div>
+
+  </template>
+  <script>
+    class LancieLogoutButton extends Polymer.Element {
```
Comment (martijnjanssen) [inline src/lancie-login/lancie-logout-button.html]: Also need to encapsulate the js of the element in a function declaration.

Comment (elarb) [reply src/lancie-login/lancie-logout-button.html]: I don't think that's necessary

**Pull request merged**